packages/wallets/src/metamask/metamaskApp.ts

```diff
@@ -0,0 +1,264 @@
+import { WalletConfig } from '../wallets.constants';
+import { WalletPage } from '../wallet.page';
+import { expect } from '@playwright/test';
+import { test, BrowserContext, Page } from '@playwright/test';
+import { MainPage, LoginPage, SettingsPage } from './pages';
+import {
+  OnboardingPage,
+  WalletOperationPage,
+  Header,
+  NetworkList,
+  OptionsMenu,
+  PopoverElements,
+  AccountMenu,
+} from './pages/elements';
+
+export class MetamaskApp implements WalletPage {
```

Comment:
we should to change name in all of our tests. 
maybe you could keep the old name ?

Reply:
there are 2 place where we need to change. Do have other reasons why we shouldn't change name?

Reply:
Maybe someone using our class outside?))

Reply:
@itaven it can be possible?

Reply:
Okay, I decided to return `MetamaskPage` but discuss it later cause i think is `MetamaskPage` is not correct name for this class